client/src/features/app/googleMaps.js

```diff
@@ -0,0 +1,89 @@
+var googleMapsAPIKey = process.env.REACT_APP_GOOGLE_MAPS_API_KEY;
+if (!googleMapsAPIKey) throw new Error("googleMapsAPIKey environment variable required");
+
+function loadScript() {
+  var script = document.createElement('script');
+  script.type = 'text/javascript';
+  script.src = 'https://maps.googleapis.com/maps/api/js?key='+googleMapsAPIKey+'&libraries=places&callback=googleMapsLoaded'; //& needed
+  document.body.appendChild(script);
+}
+
+window.googleMapsLoaded = () => {
+  initMap();
+}
+
+var map;
+var infowindow;
+var marker;
+
+function initMap() {
+  const google = window.google;
+  var position = {lat: 43.639217, lng: -79.400414};
+
+  map = new google.maps.Map(document.getElementById('map'), {
```

Comment:
did you change this line when you changed it to className? I think the 'Uncaught TypeError: Cannot read property 'firstChild' of null' error is because its still getting id map!

Reply:
You're right, Zoey, changing it to getElementByClassName worked. I missed that part. Thanks!